src/api.coffee

```diff
@@ -113,12 +113,15 @@ module.exports = (application) ->
 					.finally ->
 						application.start(app)
 		.catch (err) ->
+			status = 503
+			if err instanceof utils.AppNotFoundError
+				errMsg = "App not found: an app needs to be installed for purge to work.
+					If you've recently moved this device from another app,
+					please push an app and wait for it to be installed first."
+				err = new Error(errMsg)
```

Comment:
Well we'll only hit this error in `utils.getKnexApp` so it's pretty easy to know where it comes from, and it general it relates to the `appId` being invalid. Also this is already merged...